Source/diablo.cpp

```diff
@@ -144,6 +145,8 @@ static void diablo_parse_flags(int argc, char **argv)
 			SetPrefPath(argv[++i]);
 		} else if (strcasecmp("--config-dir", argv[i]) == 0) {
 			SetConfigPath(argv[++i]);
+		} else if (strcasecmp("--ttf-file", argv[i]) == 0) {

```

Comment:
The command is `--font-file`, but sets the `font path`, which one was intended?

Reply:
I originally set it up to do the path but I could not get the TTF_FONT_NAME to append to the path so I changed it to file. But I guess I should change the variable from ttfPath to ttfFile?

Reply:
If you make it specify an exact font we can get rid of the build time font options. So I guess that would be preferred.

Reply:
It looks like its only the amiga that uses a different font. So would we just do an #ifdef that sets it for the amiga otherwise use charissilb?

Reply:
Ok, if you think that's better then make it `--font-dir`